include/rocksdb/filter_policy.h

```diff
@@ -230,7 +238,7 @@ class FilterPolicy {
 // FilterPolicy (like NewBloomFilterPolicy) that does not ignore
 // trailing spaces in keys.
 extern const FilterPolicy* NewBloomFilterPolicy(
-    double bits_per_key, bool use_block_based_builder = false);
+    double bits_per_key, bool IGNORED_use_block_based_builder = false);
```

Comment:
Should you create a separate signature that does not have the ignored flag and have this signature just call the other one?  That will allow this API to be deprecate in the future.

Reply:
I agree the function should be deprecated but we need to come up with the alternative.

Reply:
Future work that doesn't need to be in a major release.